WiMigrator/CommandLine.cs

```diff
@@ -36,14 +36,14 @@ private void ConfigureCommandLineParserWithOptions()
                 " from one Visual Studio Team Services account to another.";
 
             CommandOption validate = commandLineApplication.Option(
-                "-v | --validate <configurationfilename>",
+                "-ver | --validate <configurationfilename>",
```

Comment:
`ver` might not make sense as an alias for `validate`. Maybe it it collides, just drop the short form entirely

Reply:
I agree with @nschonni just drop the -v and -m support.

Reply:
Sorry, I had verification in mind ;-), so val would be much better.
But as you suggest, I will remove -v and -m completely.
Thankx!